sdk/identity/azure_identity/src/client_certificate_credential.rs

```diff
@@ -84,15 +78,14 @@ impl ClientCertificateCredential {
         client_id: String,
         client_certificate: C,
         client_certificate_pass: P,
```

Comment:
Unrelated, but while you're in here:

```suggestion
        client_certificate_password: P,
```